src/book/prices/PriceDB.ts

```diff
@@ -127,8 +127,11 @@ async function getCurrencyQuotes(): Promise<Price[]> {
 }
 
 async function getInvestmentQuotes(): Promise<Price[]> {
-  const commodities = await Commodity.findBy({
-    namespace: Not('CURRENCY'),
+  const commodities = await Commodity.find({
+    where: [
+      { namespace: 'STOCK' },
+      { namespace: 'FUND' },
+    ],
```

Comment:
We used to assume that anything not 'CURRENCY' can be queried in yahoo finance but this is not the case. We will need to work in the future on this